client/src/pages/ForumPage/ForumPageDiscover.js

```diff
@@ -20,7 +23,32 @@ const handleClick = e => {
   console.log("Clicked");
 };
 
-const ForumPageDiscover = props => {
+
+class ForumPageDiscover extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      hashtags: [],
+    }
+  }
+
+  componentDidMount() {
+  axios.get(`${BASE_URL}/hashtags`)
+    .then(response => {
+      this.setState({ 
+        hashtags: response.data
+      })
+      console.log(response.data);
```

Comment:
Can you remove this and any other `console.log` statements?

Reply:
Removed console.logs from the page